mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -230,6 +355,26 @@ resource "google_bigtable_instance" "instance" {
 `, instanceName, instanceName, numNodes)
 }
 
+
+func testAccBigtableInstance_noNumNodes(instanceName string) string {
```

Comment:
Interesting! I thought this is invalid. How can this work? Will users be confused?

Reply:
This is invalid at create time but ok for update. We plan to keep this behavior in case people have config relying on this.